### ISSUE TITLE
nothing else,thanks for your code

### ISSUE THREAD

**Issue by JunlinHan** [dlam4h/OCD-PyTorch#2]
,

**Comment by harshitsankhla** [comment]
@JunlinHan If possible, can you please share the dataset you used for training ? Also which code did you try, 1 or 2 ?

**Comment by JunlinHan** [comment]
> @JunlinHan If possible, can you please share the dataset you used for training ? Also which code did you try, 1 or 2 ?

Hi, I used both PASCAL  VOC and BSDS500. From my perspective, the second one performs better.

**Comment by harshitsankhla** [comment]
Do you train with the dataset images as they are or do some pre-processing. I basically wanted to know if I should black out all the non-contour pixels or just leave them as they are in the segmentation mask.

**Comment by JunlinHan** [comment]
> Do you train with the dataset images as they are or do some pre-processing. I basically wanted to know if I should black out all the non-contour pixels or just leave them as they are in the segmentation mask.

You should leave them. They are groundtruth hence there's no need to change them.

**Comment by harshitsankhla** [comment]
I see. Thanks for the information.
